api/add_dimension.md

```diff
@@ -133,13 +106,13 @@ partitionining on `device_id`.
 
 ```sql
 SELECT create_hypertable('conditions', by_range('time'));
-SELECT add_dimension('conditions', , by_hash('location', 2));
+SELECT add_dimension('conditions', by_range('time'), by_hash('location', 2));
```

Comment:
Same here as above. And, the changes a few lines below.

Reply:
Updated.